sympy/ntheory/tests/test_factor_.py

```diff
@@ -606,3 +606,12 @@ def test_is_deficient():
     assert is_deficient(56) is False
     assert is_deficient(20) is False
     assert is_deficient(36) is False
+
+
+def test_is_amicable():
+    assert is_amicable(173, 129) is False
```

Comment:
A True, False and a test for when the numbers are the same is sufficient. So just 3 tests.